packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -107,7 +107,7 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       speak: 'none',
       verticalAlign: 'middle',
 
-      ...(!isFontBased && { boxSizing: 'border-box' }),
+      ...(isFontBased && { boxSizing: 'content-box' }),
```

Comment:
would suggest to make it part of the `getFontStyles` result - by putting it near the following related props:

```js
return {
    ...

    boxSizing: 'content-box',
    width: sizeInRems,
    height: sizeInRems,

   ...
  }
```